test/test_util.jl

```diff
@@ -50,23 +50,58 @@ function rrule_test(f, ȳ, (x, x̄)::Tuple{Any, Any}; rtol=1e-9, atol=1e-9, fdm
     test_accumulation(Zero(), dx, ȳ, x̄_ad)
 end
 
+function _make_fdm_call(fdm, f, ȳ, xs, ignores)
```

Comment:
Eventually what this function does should be moved to FDM, but it took less time to write this than it would to figure out how to incorporate it into the FDM framework directly.